calendar-bundle/contao/classes/Events.php

```diff
@@ -94,7 +94,7 @@ protected function getAllEvents($arrCalendars, $intStart, $intEnd, $blnFeatured
 	{
 		$calendarEventsGenerator = System::getContainer()->get('contao_calendar.generator.calendar_events');
 
-		return $this->arrEvents = $calendarEventsGenerator->getAllEvents($arrCalendars, (new \DateTime())->setTimestamp($intStart), (new \DateTime())->setTimestamp($intEnd), $blnFeatured, $this->cal_noSpan);
+		return $this->arrEvents = $calendarEventsGenerator->getAllEvents($arrCalendars, (new \DateTime())->setTimestamp($intStart), (new \DateTime())->setTimestamp($intEnd), $blnFeatured, $this->cal_noSpan, module: $this);
```

Comment:
```suggestion
		return $this->arrEvents = $calendarEventsGenerator->getAllEvents($arrCalendars, (new \DateTime())->setTimestamp($intStart), (new \DateTime())->setTimestamp($intEnd), $blnFeatured, $this->cal_noSpan, $this);
```

Reply:
This would lead to an exception. We need to pass it to the `$module` argument.

Reply:
ah sorry, I thought that's the one. I'm not sure we're actually using named parameters in our (legacy) code.

Reply:
```suggestion
		return $this->arrEvents = $calendarEventsGenerator->getAllEvents($arrCalendars, (new \DateTime())->setTimestamp($intStart), (new \DateTime())->setTimestamp($intEnd), $blnFeatured, $this->cal_noSpan, null, $this);
```

This should work without named parameters, shouldn't it?

Reply:
Yes, but I prefer named parameters for such cases, as that is more descriptive and safe. But will change nevertheless.